cmd/crank/beta/trace/internal/printer/default.go

```diff
@@ -216,6 +216,9 @@ func getResourceStatus(r *resource.Resource, name string, wide bool) fmt.Stringe
 	syncedCond := r.GetCondition(xpv1.TypeSynced)
 	var status, m string
 	switch {
+	case r.Unstructured.GetDeletionTimestamp() != nil:
+		// Report the status as deleted if the resource is being deleted
+		status = "Deleted"
```

Comment:
Maybe `Deleting`? The resource still existing implies something is still processing the delete.

Reply:
made the changes